### ISSUE TITLE
Textfield with error state has incorrect focus indicator

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#750]
### Expected Behaviour
``` 
<sp-textfield
                label="Name"
                placeholder="Enter your name"
                invalid
            ></sp-textfield>
```
should have a blue border when it gets keyboard focus

The correct version can be seen in https://spectrum.adobe.com/static/XD/Items/Text-field/Text-field-v5.2.0.xd

### Actual Behaviour
It gets a thicker red border instead

### Reproduce Scenario (including but not limited to)
Run https://opensource.adobe.com/spectrum-web-components/components/textfield and focus the invalid field using the keyboard

#### Steps to Reproduce
Run https://opensource.adobe.com/spectrum-web-components/components/textfield and focus the invalid field using the keyboard

#### Platform and Version
Reproduces using Safari and Chrome on MacOS

#### Sample Code that illustrates the problem
``` 
<sp-textfield
                label="Name"
                placeholder="Enter your name"
                invalid
            ></sp-textfield>
```

**Comment by jnurthen** [comment]
### Current style
![Screen Shot showing focus style in current textfield component showing red border](https://user-images.githubusercontent.com/1724479/87099248-1ff18f80-c1fe-11ea-853c-9c43a05062f1.png)
### Desired focused style (from XD design doc)
![Screen Shot of keyboard focus style from XD doc showing blue border](https://user-images.githubusercontent.com/1724479/87099323-40214e80-c1fe-11ea-967d-99d948bd615a.png)

**Comment by Westbrook** [comment]
The update to the Beta release of Spectrum CSS in #543 is looking like it will address this. Should have a PR into the feature branch where we're tracking these updates (#754) soon.
